doc/source/release-notes/Percona-Server-5.7.31-34.rst

```diff
@@ -0,0 +1,49 @@
+.. _PS-5.7.31-34:
+
+================================================================================
+*Percona Server for MySQL* 5.7.31-34
+================================================================================
+
+:Date: August 24, 2020
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/5.7/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 5.7.31-34
+includes all the features and bug fixes available in
+`MySQL 5.7.31 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-31.html>`_
+in addition to enterprise-grade features developed by Percona.
+
+New Features
+================================================================================
+
+* :jirabug:`PS-7128`: Document RocksDB variables: rocksdb_max_background_compactions, rocksdb_max_background_flushes, and rocksdb_max_bottom_pri_background_compactions
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7132`: Make default value of rocksdb_wal_recovery_mode compatible with InnoDB
+* :jirabug:`PS-7199`: Document Coredumper functionality
+* :jirabug:`PS-7114`: Enhance crash artifacts (core dumps and stack traces) to provide additional information to the operator
```

Comment:
should this be "Implemented Coredumper funcionality", with link to feature documentation/description.